src/Worker.cs

```diff
@@ -31,10 +31,48 @@ public async static Task Main(string[] args)
                 LogLevel.Information,
                 string.Format(PowerShellWorkerStrings.PowerShellWorkerVersion, typeof(Worker).Assembly.GetName().Version));
 
-            WorkerArguments arguments = null;
-            Parser.Default.ParseArguments<WorkerArguments>(args)
-                .WithParsed(ops => arguments = ops)
-                .WithNotParsed(err => Environment.Exit(1));
+            var workerOptions = new WorkerOptions();
+
+            var parser = new Parser(settings => settings.EnableDashDash = true);
```

Comment:
@Francisco-Gamino isn't there an property you can set using the settings provided here to allow unknown arguments? I was under the impression that was the case, and if so, setting that may simplify your change a bit.

Reply:
Thank you @fabiocav for the suggestion. I have updated the parser settings to enable `IgnoreUnknownArguments`. I am also setting `EnableDashDash` which enables double dash '--' syntax to force parsing of all subsequent tokens as values.